SimDataFormats/Associations/plugins/MCToCPAssociatorEDProducer.cc

```diff
@@ -0,0 +1,94 @@
+//
```

Comment:
this seems to grow  SimDataFormats/Associations/plugins.

@cms-sw/core-l2 is this a supported case or is there a chance for problems to have /plugins in a DataFormats?

Reply:
> is this a supported case or is there a chance for problems to have /plugins in a DataFormats?

It is not, plugins are not allowed in DataFormats (as mentioned in https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#Producing_the_EDProduct
> **NOTE: The producer must not be in the same package as the EDProduct.**

)

Reply:
@makortel shall I move it here?
https://github.com/cms-sw/cmssw/tree/716967537b696c443b4a095f0008902ee692dc9f/SimCalorimetry/HGCalAssociatorProducers/plugins

Reply:
@lecriste By quick look that looks like a good place, thanks.

Reply:
OK, I moved the plugin introduced by this PR.
Will move the rest in another PR soon.